hw11/css/styles_part1.css

```diff
@@ -0,0 +1,86 @@
+/* Need this to prevent vertical scroll since body takes 100vh */
+* {
+  padding: 0;
+  margin: 0;
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+}
+
+.page {
+  width: 100vw;
+  min-height: 100vh;
+  background-color: #ffffff;
+}
+
+.page-area {
+  width: 90%;
+  height: 90%;
+  margin: 5%;
+  padding: 5%;
+  -webkit-box-shadow: 0 4px 68px rgba(0, 0, 0, 0.1);
+  box-shadow: 0 4px 68px rgba(0, 0, 0, 0.1);
+  background-color: #ffffff;
+}
+
+.page-header {
+  color: #171717;
+  font-family: "Playfair Display", serif;
+  font-size: 42px;
+  font-weight: 400;
+  letter-spacing: -0.21px;
+}
+
+.capital-letter {
+  color: #171717;
+  font-family: "Playfair Display", serif;
+  font-size: 100px;
+  font-weight: 400;
+  text-transform: uppercase;
+  letter-spacing: -0.55px;
+  margin: -10px 15px -20px 0;
+  float: left;
```

Comment:
Be careful with `float`, since floated elements are not available in the normal document flow. And parent needs to be 'clearfixed'.

Reply:
Fixed.